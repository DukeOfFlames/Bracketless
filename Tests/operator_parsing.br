START

{
    fn f{} { -> 2 }
    fn g{} { -> 3 }
    #drucke{f{} + g{}}
}

{
    #drucke{2 + 2 + 3}
}

{
    °a = 3
    °b = 4
    #drucke{a! + b!}
}

{
    °a = true
    °b = false
    #drucke{not a and not b}
}

{
    °a = 3
    #drucke{a!!}
}

{
    °a = true
    #drucke{not not a}
}

END