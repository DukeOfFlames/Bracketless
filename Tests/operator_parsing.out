5
7
30
false
720
true
